index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
You should add an HTML Title tag </title> .
Title tags are a very important part of on-page search engine optimization, so it's crucial to use them properly.